nowcasting_utils/models/losses/StructuralSimilarity.py

```diff
@@ -0,0 +1,78 @@
+import torch
+from pytorch_msssim import SSIM, MS_SSIM
+from torch import nn as nn
+
+
+class SSIMLoss(nn.Module):
```

Comment:
Please add a brief docstring (even if it just says that SSIM stands for Structural Similarity Index Measure, maybe with a link to https://en.wikipedia.org/wiki/Structural_similarity :)